rover_shared_libs/rover_lib2/src/rover_lib2/helpers/constants.hpp

```
@@ -40,6 +60,16 @@ namespace Constants
          * @return Success on camera name found
          */
         bool getNameFromURL(const std::string& url_, std::string& rName_);
+        eCamNames getIndexFromName(const std::string& name_);
```

Remove this helper and force other packages to not use the name as an input as it can fail silently. Force them to use the enum directly